runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -119,6 +120,20 @@ private void waitForWorkflow(@NonNull UUID workflow, @NonNull ManagementClient c
         throw new TimeoutException();
     }
 
+    /*
+     * private method to handle exceptions while sendRequest()
+     */
+    private void handleException(RuntimeException runtimeException) {
+        final Throwable cause = ExceptionUtils.getRootCause(runtimeException);
+        if (cause instanceof TimeoutException ||
+                cause instanceof IllegalStateException ||
+                cause instanceof NetworkException) {
+            log.warn("WorkflowRequest: Error while running {} with cause:", this, runtimeException);
```

Comment:
another curly bracket needed for runtimeException

Reply:
Unfortunately, you are not correct, another bracket is not needed